src/adapters/MeetingsJSONAdapter.js

```diff
@@ -436,7 +446,12 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
       }
     }
 
-    return {stream, perm, deviceId};
+    return {
+      stream,
+      perm,
+      deviceId,
+      err,
```

Comment:
this error should be read by getMeeting() and put in meeting.localVideo.error so that it can be seen by the mute video control.